docs/01-start-here/01-installation-steps.md

```diff
@@ -18,12 +18,14 @@ All development laptops must be encrypted. If you are not 100% sure, please ask
 Follow [this link](https://www.google.co.uk) and enter the relevant search string for your system.
 
 # Obtain AWS credentials
-1. Fork [Janus](https://github.com/guardian/janus) and follow the readme
+1. Fork [Janus](https://github.com/guardian/janus) and follow the readme (**Note:** you will need permission to access the Janus repo to do this, and 2FA set up on your Google account)
 2. Make your change and push to a new branch (you can review the closed PRs for help)
 3. Submit a PR
 
 # Local Test Server setup
 
+You can either set up the frontend using a Docker container, or through a local install on your machine. Before checking out the repository you may need to [add an SSH key to your GitHub account](https://help.github.com/articles/generating-ssh-keys/). Before pushing changes you may need to [create an access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/).
```

Comment:
@TBonnin we should probably adding a cautionary message re: docker here?

(not for this PR though probably)